### DISCUSSION TITLE
`@import 'my.user.css'` results in that browser tries to download local file from the server, not the folder of my computer :(

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [openstyles/stylus#1831] [Q&A]
Hi, is it possible to use `@import 'something.css'`, to have multiple files in one folder?

![image](https://github.com/user-attachments/assets/8370aec1-b8fe-422a-9f7a-6d2fe0bf9d1c)

**Comment by tophf** [comment]
It's not implemented, see #465.
